README.md

```diff
@@ -9,31 +9,63 @@ view-git-file macro on any Confluence page.
 ## Marketplace
 
 You can install Git for Confluence by using the Universal Plugin Manager inside
-Confluence, or by visiting the [Git for Confluence marketplace entry][1]
+Confluence, or by visiting the
+[Git for Confluence marketplace entry](https://avisi-apps.gitbook.io/git-for-confluence/.md).
 
 ## Documentation
 
 If you want to learn how to configure Git for Confluence or use the Git for
-Confluence macro's, please refer tot the [Git for Confluence documentation][2]
+Confluence macro's, please refer tot the
+[Git for Confluence documentation](https://avisi-apps.gitbook.io/git-for-confluence/).
 
 # Formats
 
+The are three main classes of formats that the Git for Confluence add-on natively
+supports. These consist of Markdown, Image and PlantUML files. Showcase examples
+that demonstrate how these can be used and rendered in Confluence are available
+in the formats directory:
+```bash
+.
+├── README.md
```

Comment:
Dit is wel onderhoudsgevoelig

Reply:
Zeker, naar mijn mening worth it omdat we de repo hopelijk niet vaak gaan updaten.